src/goto/storage.py

```diff
@@ -304,3 +310,26 @@ def get_prefix_expansions(prefix):
     subprefixes = list_subprefixes(prefix)
     teleport_no_prefix, _ = os.path.split(prefix)
     return [os.path.join(teleport_no_prefix, x) + os.sep for x in subprefixes]
+
+def set_config(attr, value):
+    '''sets attr to value'''
+    data = _get_settings()
+    data[attr] = int(value)
```

Comment:
Just putting in `value` directly is fine here. No need to convert to int. However, I do want a whitelist for this. So for example you can have a global `ALLOWED_SETTING_KEYS = set(['test_key'])` and check `if not attr in ALLOWED_SETTING_KEYS: ... raise StorageException(msg)`

Reply:
I'll change these and get back asap.